ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/route53/provision/cucushift-installer-rehearse-aws-ipi-route53-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-route53-provision
+  steps:
+    - ref: ipi-conf
+    - ref: aws-provision-vpc-shared
+    - ref: aws-provision-bastionhost
+    - ref: ipi-conf-aws
```

Comment:
as comment in https://github.com/openshift/release/pull/27656/files#r846992212, the step `ipi-conf-aws-custom-vpc` will be required before `ipi-conf-aws`

Reply:
Done